timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResults.vue

```diff
@@ -274,18 +300,30 @@ export default {
       if (sessionQueue.length > 0 && !this.interval) {
         this.interval = setInterval(
           function () {
-            if (sessionQueue.length > 0) {
-              // fetch data for sessions in the queue
-              this.fetchAnalyzerSessionData()
-              // update active session queue
-              this.fetchActiveSessions()
-            } else {
+            if (sessionQueue.length === 0 || this.activeAnalyzerQueue.length === 0) {
               // the queue is empty so stop the interval
+              clearInterval(this.interval)
+              this.activeAnalyzerTimerStart = null
+              this.interval = false
+              return
+            }
+            // check timeout
+            if (this.activeAnalyzerTimerStart !== null && (Date.now() - this.activeAnalyzerTimerStart > this.activeAnalyzerTimeout)) {
               clearInterval(this.interval)
               this.interval = false
+              this.activeAnalyzerTimerStart = null
+              this.activeAnalyzerTimeoutTriggered = true
+              return
             }
+            // set dynamic interval
+            if (sessionQueue.length >= 10) this.activeAnalyzerInterval = 30000
+            if (sessionQueue.length >= 50) this.activeAnalyzerInterval = 60000
```

Comment:
I know this is a stopgap solution before we write the new API endpoint. Please add a comment here mentioning that and a TODO to remove this later